src/cryptography/hazmat/primitives/padding.py

```diff
@@ -122,3 +126,53 @@ def finalize(self):
         res = self._buffer[:-pad_size]
         self._buffer = None
         return res
+
+
+class PKCS7(_BytePadding):
```

Comment:
You're still using subclassing here (and for ANSIX923). Could you please remove the subclassing and instead make the size checks from the parent class's `__init__` be a function you call?